app/src/main/java/com/infomaniak/mail/utils/FetchMessagesManager.kt

```diff
@@ -97,23 +120,25 @@ class FetchMessagesManager @Inject constructor(
         threadController.fetchMessagesHeavyData(messages, mailbox, realm, okHttpClient)
 
         val message = MessageController.getThreadLastMessageInFolder(uid, realm) ?: run {
-            ThreadController.getThread(uid, realm)?.let { thread ->
-                Sentry.withScope { scope ->
-                    scope.level = SentryLevel.ERROR
-                    scope.setExtra("does Thread still exist ?", "[true]")
-                    scope.setExtra("currentMailboxEmail", "[${AccountUtils.currentMailboxEmail}]")
-                    scope.setExtra("mailbox.email", "[${mailbox.email}]")
-                    scope.setExtra("messageUid", "[$sentryMessageUid]")
-                    scope.setExtra("folderName", "[${thread.folder.name}]")
-                    scope.setExtra("threadUid", "[${thread.uid}]")
-                    scope.setExtra("messagesCount", "[${thread.messages.count()}]")
-                    scope.setExtra("messagesFolder", "[${thread.messages.map { "${it.folder.name} (${it.folderId})" }}]")
-                    Sentry.captureMessage("We are supposed to display a Notification, but we couldn't find the Message in the Thread.")
-                }
-            }
+            SentryDebug.sendFailedNotification(
+                "Can't find the Message in the Thread to show the Notification",
```

Comment:
Here as well we could replace the four places that start with "Can't find the Message ..." with something more easily readable like "Message not found ..."